inputs/README.md

```diff
@@ -6,4 +6,8 @@ To download all input data at once you can run the following command (from the b
 $ python innovation_sweet_spots/getters/inputs.py
 ```
 
-To download specific input datasets, see the documentation in `inputs.py`
+To download specific input datasets, see the documentation in `inputs.py`. Presently, the datasets available via `inputs.py` are the following:
+
+- GTR research projects (approximately 400 MB)
+- Crunchbase organisations (<1GB)
+- Hansard speeches (3+GB)
```

Comment:
There is a question about how much data you want to have on your local machine at any one time. It might be useful at the "read" stage (i.e. here) to already filter out columns or rows that aren't needed